windows/win_lineinfile.py

```diff
@@ -99,21 +99,44 @@
     default: "windows"
 """
 
-EXAMPLES = """
-- win_lineinfile: dest=C:\\temp\\example.conf regexp=^name= line="name=JohnDoe"
+EXAMPLES = r"""
+- win_lineinfile:
+    dest: C:\temp\example.conf
+    regexp: '^name='
+    line: "name=JohnDoe"
 
-- win_lineinfile: dest=C:\\temp\\example.conf state=absent regexp="^name="
+- win_lineinfile:
+    dest: C:\temp\example.conf
+    regexp: '^name='
+    state: absent
 
-- win_lineinfile: dest=C:\\temp\\example.conf regexp='^127\.0\.0\.1' line='127.0.0.1 localhost'
+- win_lineinfile:
+    dest: C:\temp\example.conf
+    regexp: '^127\.0\.0\.1'
+    line: '127.0.0.1 localhost'
 
-- win_lineinfile: dest=C:\\temp\\httpd.conf regexp="^Listen " insertafter="^#Listen " line="Listen 8080"
+- win_lineinfile:
+    dest: C:\temp\httpd.conf
+    regexp: "^Listen "
+    insertafter: "^#Listen "
+    line: "Listen 8080"
 
-- win_lineinfile: dest=C:\\temp\\services regexp="^# port for http" insertbefore="^www.*80/tcp" line="# port for http by default"
+- win_lineinfile:
+    dest: C:\temp\services
+    regexp: "^# port for http"
+    insertbefore: "^www.*80/tcp"
+    line: "# port for http by default"
 
 # Create file if it doesn't exist with a specific encoding
-- win_lineinfile: dest=C:\\temp\\utf16.txt create="yes" encoding="utf-16" line="This is a utf-16 encoded file"
+- win_lineinfile:
+    dest: C:\temp\utf16.txt
+    create: "yes"
+    encoding: "utf-16"
+    line: "This is a utf-16 encoded file"
 
 # Add a line to a file and ensure the resulting file uses unix line separators
-- win_lineinfile: dest=C:\\temp\\testfile.txt line="Line added to file" newline="unix"
-
+- win_lineinfile:
+    dest: C:\temp\testfile.txt
+    line: "Line added to file"
+    newline: "unix"
```

Comment:
We can ditch all these quotes with the k=v conversion (well, except maybe the ones with embedded =, not sure about that).